src/components/models/GoogleLoginModal.js

```diff
@@ -138,9 +142,9 @@ const GoogleLoginModal = ({
                         placeholder="Course Name"
                       >
                         <option>Select your room</option>
-                        {rooms.map((data) => {
-                          return <option>{data}</option>;
-                        })}
+                        {rooms.map((data) => (
+                          <option>{data}</option>
```

Comment:
I didn't notice before but I think we need key attribute here
Do you know which property of data we can use as unique key?

Reply:
u can use data.id